thirdparty/CMakeLists.txt

```diff
@@ -38,24 +38,24 @@ target_include_directories(inicpp INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/inifile_
 add_library(inicpp::inicpp ALIAS inicpp)
```

Comment:
Should we do EXCLUDE_FROM_ALL for add_library here too?

I'm not sure why we're doing EXCLUDE_FROM_ALL for all add_subdirectory calls, let's add a comment about this?

Reply:
added comment